test/test-genesis.ts

```diff
@@ -384,4 +385,66 @@ describe('Genesis Contract', () => {
       contract.connect(owner).mintReservedGods(whitelistedAddr, 10),
     ).to.be.revertedWith('Not enough reserved gods left');
   });
+
+  it('Contract base URI is unrevealed URI if not changed', async function () {
+    await contract.connect(owner).unpause();
+    expect(await contract.tokenURI(1)).to.be.equal(constants.unrevealedURI);
+  });
+
+  it('Contract base URI is unrevealed URI if not changed', async function () {
+    await contract.connect(owner).unpause();
+    expect(await contract.tokenURI(1)).to.be.equal(constants.unrevealedURI);
+    expect(await contract.tokenURI(100)).to.be.equal(constants.unrevealedURI);
+    expect(await contract.tokenURI(4000)).to.be.equal(constants.unrevealedURI);
+  });
+
+  it('Contract base URI is revealed URI if set', async function () {
+    await contract.connect(owner).unpause();
+    await contract.connect(owner).setBaseURI(constants.revealedURI);
+
+    expect(await contract.tokenURI(1)).to.be.equal(
+      `${constants.unrevealedURI}1`,
```

Comment:
should be `constants.revealedURI`, the test only passes cause right now `revealedURI === unrevealedURI`

Reply:
lol good catch